### ISSUE TITLE
No way to set a specific source

### ISSUE THREAD

**Issue by JH2535** [AcidWeb/CurseBreaker#330]
I thought with the new update and the removal of curseforge support we would be able to set the sources for specific addons, so for example I know all the things has a repo https://github.com/DFortun81/AllTheThings, I would like to set the source to be that. How can I do this with 4.0?

**Comment by AcidWeb** [comment]
Yes. Currently there is no nice option to switch the sources. You need to uninstall addon and install it from different source. Addon settings will not be touched.
